app/controllers/users.py

```diff
@@ -150,7 +192,7 @@ def get(self):
         
         return dict(
             status='success',
-            data=dict(pagination=pagination, users=json.loads(users_data))
+            data=dict(meta_data = meta_data , pagination=pagination, users=json.loads(users_data))
```

Comment:
Change the meta data output to only show beta_users, non_verified and total_users something like;
`{
  beta_users: 131,
  non verified: 123,
  total_users: 3223
  }
`
.
<img width="372" alt="Screenshot 2023-08-18 at 3 27 29 AM" src="https://github.com/crane-cloud/backend/assets/18049550/5112892d-689b-4f60-a396-c4d51847f561">